tests/unit/lms/services/exceptions_test.py

```diff
@@ -208,7 +208,20 @@ class TestCanvasAPIError:
                 OAuth2TokenError,
             ),
             # A permissions error from Canvas, because the Canvas user doesn't
-            # have permission to make the API call.
+            # have permission to make the API call. Updated Canvas instances
+            # will send 403, rather than 401.
+            (
+                403,
+                json.dumps(
+                    {
+                        "status": "unauthorized",
+                        "errors": [
+                            {"message": "user not authorized to perform that action"}
+                        ],
+                    }
+                ),
```

Comment:
The repeated dumps here are a bit annoying, but it's because there's one "raw" string at the end.